protocol/writing-resource/put-patch-auxiliary-resource.feature

```diff
@@ -0,0 +1,19 @@
+Feature: PUT or PATCH on auxiliary resources
+
+  Background: Set up container and parse link headers
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.createContainer()  
+    * def exampleTurtle = karate.readAsString('../fixtures/example.ttl')
+    * def rdfResource = testContainer.createResource('.ttl', exampleTurtle, 'text/turtle');
+
+  Scenario: PUT auxiliary resource to container
+    * def response = clients.alice.sendAuthorized('GET', container.url, null, null)
+    * def links = parseLinkHeaders(response.headers)
+    * def describedby = links.find(el => el.rel === 'describedBy')
```

Comment:
Is links.find case-sensitive?

Reply:
I think so. Howso?

Reply:
Comparison should be case-insensitive as per https://datatracker.ietf.org/doc/html/rfc8288#section-2.1.1 .

Reply:
Ah, right, thanks! It is then not about `links.find`, but about the actual comparison. In this case, in which the relation has known properties, we can just do `.toLowerCase()`.